src/App.js

```diff
@@ -74,9 +80,33 @@ function App() {
     }
   };
 
+  const setTitle = (titleText) => {
+    setAPIData( prevState => {
+      console.log( prevState.notificabottomButtonText);
```

Comment:
can you remove this line?

Reply:
sorry about that, I forgot it.
here is the fix: https://github.com/DynamoDS/NotificationCenter/pull/30/commits/6ab30190d950908162dd3a8e67a0f0ab2a1ea4d4